ecommerce/models_test.py

```diff
@@ -44,11 +50,49 @@ def test_order_audit(has_user):
 
     assert order_audit.data_after == {
         **serialize_model_object(order),
-        "lines": [serialize_model_object(line) for line in lines],
+        "purchaser_email": order.purchaser.email,
+        "lines": [
+            {
+                **serialize_model_object(line),
+                "product_version_info": {
+                    **serialize_model_object(line.product_version),
+                    "product_info": {
+                        **serialize_model_object(line.product_version.product),
+                        "content_type_string": str(
+                            line.product_version.product.content_type
+                        ),
+                        "content_object": serialize_model_object(
+                            line.product_version.product.content_object
+                        ),
+                    },
+                },
+            }
+            for line in lines
+        ],
         "coupons": [
-            serialize_model_object(coupon)
+            {
+                **serialize_model_object(coupon),
+                "coupon_version_info": {
+                    **serialize_model_object(coupon.coupon_version),
+                    "payment_version_info": serialize_model_object(
+                        coupon.coupon_version.payment_version
+                    ),
+                },
+            }
             for coupon in order.couponredemption_set.all()
         ],
+        "run_enrollments": [
+            enrollment.run.courseware_id
+            for enrollment in order.courserunenrollment_set.all()
+        ],
```

Comment:
I noticed that `run_enrollments` is blank in the `OrderAudit`s unless I save the `Order` again from django admin, I think this is because `order.save_and_log` is not called again after `complete_order()` in the `OrderFullfillmentView` at https://github.com/mitodl/mitxpro/blob/b44bf34913b0bdbcc26cca83a9a5ac0a2df7c7e9/ecommerce/views.py#L168-L172